opentreemap/treemap/js/src/mapFeatureDetail.js

```diff
@@ -198,39 +199,62 @@ function init() {
             .onValue(panorama.update);
     }
 
+    handleFavoriteClick();
+
+    socialMediaSharing.init({imageFinishedStream: imageFinishedStream});
+}
+
+function setFavoritedState() {
     if (config.loggedIn) {
-        $('body').on('click', dom.favoriteLink, function(e) {
-            var $favoriteLink = $(dom.favoriteLink),
-                $favoriteIcon = $(dom.favoriteIcon),
-                wasFavorited = $favoriteLink.attr('data-is-favorited') === 'True',
-                url = $favoriteLink.attr(wasFavorited ? 'data-unfavorite-url' : 'data-favorite-url');
+        updateFavoritedState(isFavoriteNow());
+    }
```

Comment:
I think the `if (config.loggedIn) {` check needs to be removed from this function.

With it in place, I do not see a tooltip when not logged in.